### ISSUE TITLE
Colorbar Label in RootInteractive plots

### ISSUE THREAD

**Issue by sadkoeni** [miranov25/RootInteractive#216]
Hi Marian, 

It currently seems impossible to set the label of the colorbar when plotting a RootInteractive histogram. 
It will greatly improve the understandibility of the dashboards if this would be possible.

**Comment by miranov25** [comment]
Hello @sadkoeni 
I agree. We will try to have it in the next Release. @pl0xz0rz, could you  add this request. i assume it will need only definition of the zAxisTitle

**Comment by miranov25** [comment]
Hello @sadkoeni and @bulukutlu 

As we quoted above there was a fix implemented in #250. In addition today we merge a new functionality for the heat map.



*https://gitlab.cern.ch/alice-tpc-offline/alice-tpc-notes/-/blob/5d0d108e0dac11c8c95544090cb951deaa552d5e/JIRA/ATO-568/kryptonAnalysisPerPad.ipynb

```python
    #    tab 6
            #[[("bin_bottom_0", "bin_top_0")], [("bin_bottom_1", "bin_top_1")], {"colorZvar": "log(bin_count+1)", "source":"histoAB"}],
    [[("bin_bottom_0", "bin_top_0")], [("bin_bottom_1", "bin_top_1")], {"yAxisTitle":"Δ Fraction predict" ,"xAxisTitle":"Fraction", "source":"his2DConvFractionRF3","colorZvar": "log(bin_count+1)"}],
    [[("bin_bottom_0", "bin_top_0")], [("bin_bottom_1", "bin_top_1")], {"yAxisTitle":"Δ Lambda predict" ,"xAxisTitle":"Lambda", "source":"his2DConvLambdaRF3","colorZvar": "log(bin_count+1)"}],
    [['bin_center_0'], ["quantile_0","mean"], {"yAxisTitle":"(mean/median) fraction IT diff." ,"xAxisTitle":"fraction IT pred.", "source":"his2DConvFractionRF3_1"}],
    [['bin_center_0'], ["quantile_0","mean"], {"yAxisTitle":"(mean/median) slope IT diff." ,"xAxisTitle":"slope IT pred.","source":"his2DConvLambdaRF3_1"}],
    [['bin_center_0'], ["std"], {"yAxisTitle":"(RMS) fraction IT diff." ,"xAxisTitle":"fraction IT pred.","source":"his2DConvFractionRF3_1"}],
    [['bin_center_0'], ["std"], {"yAxisTitle":"(RMS) slope IT diff." ,"xAxisTitle":"slope IT pred.","source":"his2DConvLambdaRF3_1"}],
    [["pullConvFractionRF3"], ["hisPullConvFractionRF3"]],
    [["pullConvLambdaRF3"], ["hisPullConvLambdaRF3"]],

    figureGlobalOption
]


```
